docs/getting-started/users-page.asciidoc

```diff
@@ -0,0 +1,30 @@
+[[users-page]]
+= Users
+
+The Users page provides a comprehensive overview of metric user data to understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information you can view:
```

Comment:
```suggestion
The Users page provides a comprehensive overview of metric user data to understand authentication and usage information within your environment. Key Performance Indicator (KPI) charts, data tables, and interactive widgets let you view specific data and drill down for deeper insights. Here's some of the information available to you:
```

Reply:
Mainly making this rec to avoid using "view".

Reply:
* +1 @nastasha-solomon's comment, and/or maybe shorten to "Here's some of the information available:"

* Does "Key Performance Indicator" need to be capitalized? AFAIK it's a general term.